src/main/resources/lib/login.js

```diff
@@ -226,7 +232,17 @@ function updateUserData(claims, idProviderConfig, user) {
         }));
 
         log.debug(`User [${user.key}] updated with displayName [${displayName}] and email [${email}]`);
+
+        sendUserEvent(idProviderConfig.userEventPrefix, 'modify', user);
     }
 }
 
+function sendUserEvent(eventPrefix, eventType, user) {
+    eventLib.send({
+        type: `${eventPrefix}.user.${eventType}`,
+        distributed: true,
```

Comment:
it should be possible via config to disable events and optionally make them distributable:
`userEventMode : off | local | distributed